.changeset/smart-goats-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/contracts-bedrock': patch
+---
+
+bumps sdk version to have access to sepolia deployments
```

Comment:
does it make sense to add a changeset to contracts-bedrock? I could see either choice

Reply:
yea, i only bumped it because the gateway includes it as a dependency and i couldn't get the imports to resolve otherwise